manifests/y/yuma14/ShutdownCountdown/1.1.0/yuma14.ShutdownCountdown.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: yuma14.ShutdownCountdown
+PackageVersion: 1.1.0

```

```suggestion
PackageVersion: 1.1.0
AppsAndFeaturesEntries:
- DisplayVersion: 1.0.0
```
The package is marketed as `1.1.0` but the `DisplayVersion` written to Registry (that you can see in Control Panel / Apps&Features) is `1.0.0`. If this is a mistake with the installer, then please fix this behavior and update the manifest accordingly as WinGet requires PackageVersion to match with DisplayVersion for matching upgrades.

If not, you can use AppsAndFeaturesEntries as I've suggested here to explicitly provide DisplayVersion `1.0.0` to help WinGet match upgrades and use `1.1.0` (Marketing version) for PackageVersion field that users will see with `winget install yuma14.ShutdownCountdown`. That being said - I'd strongly recommend that the installer be updated to write the correct version to Registry instead of going the AppsAndFeaturesEntries route.